### DISCUSSION TITLE
Configuration Docker File

### DISCUSSION THREAD

**Discussion by stevenbaert** [open-webui/open-webui#4756] [Q&A]
Hi,

I have a yaml file of which some settings I am not sure and could not find in the documentation.
I use these but could you shed a light for correctness/context of usage?

Thanks!
S.
      - USE_EMBEDDING_MODEL_DOCKER=sentence-transformers/all-MiniLM-L6-v2
      - WEBUI_SECRET_KEY=''
      - RAG_RERANKING_MODEL=sentence-transformers/all-MiniLM-L6-v2
      - USE_RERANKING_MODEL_=true

**Comment by dannystaple** [comment]
Do you mean a docker-compose yaml file? Or if not, what YAML file is it?
I see RAG_RERANKING_MODEL and WEBUI_SECRET_KEY as environment variables documented at https://docs.openwebui.com/getting-started/env-configuration. I don't know about the other variables. I am new to Open Web UI too, but I hope that helps a little.

**Reply by stevenbaert** [reply]
Yaml docker indeed 
I was referring to that link as not able to find default values on that link.
